merlin/study/celeryadapter.py

```diff
@@ -229,7 +229,7 @@ def start_celery_workers(spec, steps, celery_args, just_return_command):
             if not check_machines(worker_machines):
                 continue
 
-            if yenv:
+            elif yenv:
```

Comment:
why this change? does it matter?

Reply:
It looks like it won't change the logic since its conditional predecessor is a `continue`.

Reply:
The ```if yenv``` is actually separate logic from the continue on the check machines.

Reply:
Uh oh I was wrong, it looks like we should indeed revert this.

Reply:
You weren't wrong Ben, the logic is equivalent, this is just a trivial style preference I have. I'll revert it because that seems to be what the team prefers :)